packaging/files/profile.deb

```diff
@@ -0,0 +1,2 @@
+export GEM_PATH="/opt/ood/gems:${GEM_PATH:+:${GEM_PATH}}"
```

Comment:
Laughably, I think my confusion in the other thread is the name of this file is `profile` implying that it replaced a profile file.

Can we rename this to `enable.deb`?

Reply:
I renamed the file to `enable.deb`